controllers/v1/users.js

```diff
@@ -584,6 +584,49 @@
            }
          });
      }
+
+           /**
+     * @api {get} /kendra/api/v1/users/listStats List All Stats
+     * @apiVersion 1.0.0
+     * @apiName List All Stats
+     * @apiGroup Users
+     * @apiHeader {String} X-authenticated-user-token Authenticity token
+     * @apiSampleRequest /kendra/api/v1/users/listStats
+     * @apiUse successBody
+     * @apiUse errorBody
+     * @apiParamExample {json} Response:
+     */
+ 
+      /**
+    * List All Stats
+    * @method
+    * @name listStats
+    * @param {Object} req - requested data.
+    * @param {String} req.query.type - requested type
+    * @returns {Object} list of stats of the user using overall program 
+    */
+ 
+     async listStats(req) {
```

Comment:
controller name can be userOverview @borkarsaish65

Reply:
done